api/app/controllers/folders_controller.rb

```diff
@@ -1,8 +1,6 @@
 class FoldersController < ApplicationController
-  # before_action :authenticate_user!, except: [:index, :show]
-  before_action :authenticate_user!, only: [ :my_folders ] # temporary until users
-  # before_action :authorize_user!, only: [ :update, :destroy, :add_sound, :remove_sound ]
-  before_action :set_current_user
+  before_action :authenticate_user!, except: [:index, :show, :folder_slug_list, :get_name]
+  before_action :authorize_user!, only: [ :update, :destroy, :add_sound, :remove_sound ]
```

Comment:
Take it or leave it, but I would probably make the except and only: blocks consistent between these. Like use only: or except: for both, not one for each. If that makes one of the list of actions way too long then no worries